Zend/zend_ast.c

```diff
@@ -1189,6 +1204,8 @@ ZEND_API void ZEND_FASTCALL zend_ast_destroy(zend_ast *ast)
 		}
 	} else if (EXPECTED(ast->kind == ZEND_AST_CONSTANT)) {
 		zend_string_release_ex(zend_ast_get_constant_name(ast), 0);
+	} else if (EXPECTED(ast->kind == ZEND_AST_OP_ARRAY)) {
+		zval_ptr_dtor_nogc(&((zend_ast_zval*)(ast))->val);
```

Comment:
Nit: This is a useless call, really. Maybe just `ZEND_ASSERT(!Z_REFCOUNTED_P(&((zend_ast_zval*)(ast))->val))`?